<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="NameGruop_companyName_etc.TestComponents.Listeners"/>
  </listeners>
  <test thread-count="5" parallel="tests" name="Error Validations(failed)">
    <classes>
      <class name="NameGruop_companyName_etc.Test.ErrorValidations">
        <methods>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="loginErrorValidation"/>
          <include name="ProductErrorValidation"/>
        </methods>
      </class> <!-- NameGruop_companyName_etc.Test.ErrorValidations -->
    </classes>
  </test> <!-- Error Validations(failed) -->
  <test thread-count="5" parallel="tests" name="Submit Order Test(failed)">
    <classes>
      <class name="NameGruop_companyName_etc.Test.SubmitOrderTest">
        <methods>
          <include name="submitOrder" invocation-numbers="0 1"/>
          <include name="launchApplication"/>
          <include name="tearDown"/>
          <include name="OrderHistoryTest"/>
        </methods>
      </class> <!-- NameGruop_companyName_etc.Test.SubmitOrderTest -->
    </classes>
  </test> <!-- Submit Order Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
